x/ratelimit/keeper/msg_server_test.go

```diff
@@ -41,6 +41,10 @@ func (s *KeeperTestSuite) TestMsgServer_AddRateLimit() {
 
 	denom := addRateLimitMsg.Denom
 	channelId := addRateLimitMsg.ChannelId
+	channelValue := int64(100)
+
+	// Mint tokens for generating channel value
+	s.App.BankKeeper.MintCoins(s.Ctx, minttypes.ModuleName, sdk.NewCoins(sdk.NewInt64Coin(addRateLimitMsg.Denom, channelValue)))
```

Comment:
can you grab and check the error that's returned from all these?

Reply:
Sure. I will.